public/API/API_GetClaims.php

```diff
@@ -1,14 +1,13 @@
 <?php
 
-use App\Community\Enums\ClaimFilters;
-use Illuminate\Support\Facades\Validator;
-use Illuminate\Validation\Rule;
-
 /*
  *  API_GetClaims - returns information about 1000 max set claims, sorted by latest `Created` date.
+ *    o : offset - number of entries to skip (default: 0)
+ *    c : count - number of entries to return (default: 100, max: 500)
```

Comment:
The comment on line 4 says this returns a maximum of 1000 claims (which I don't think it was actually enforcing)

Reply:
I didn't even see this comment.

Given that it's already there, maybe a happy medium is to not support pagination, but just cap the response correctly to 1000 results.

I've gone in this direction in latest. Now the response shape is exactly the same, but ~10x smaller in size.